config/schemas/config.yaml

```diff
@@ -4061,6 +4061,124 @@ properties:
+          configReloaders:
+            $ref: '#/properties/alerts/properties/runbookUrls/$defs/upstreamRunbook'
+          coreDns:
+            $ref: '#/properties/alerts/properties/runbookUrls/$defs/noUpstreamRunbook'
+          dailyChecks:
+            $ref: '#/properties/alerts/properties/runbookUrls/$defs/noUpstreamRunbook'
+          diskPerf:
+            $ref: '#/properties/alerts/properties/runbookUrls/$defs/noUpstreamRunbook'
+          falco:
+            $ref: '#/properties/alerts/properties/runbookUrls/$defs/noUpstreamRunbook'
+          fluentd:
+            $ref: '#/properties/alerts/properties/runbookUrls/$defs/noUpstreamRunbook'
+          general:
+            $ref: '#/properties/alerts/properties/runbookUrls/$defs/upstreamRunbook'
+          harbor:
+            $ref: '#/properties/alerts/properties/runbookUrls/$defs/noUpstreamRunbook'
+          hnc:
+            $ref: '#/properties/alerts/properties/runbookUrls/$defs/noUpstreamRunbook'
+          kubeStateMetrics:
+            $ref: '#/properties/alerts/properties/runbookUrls/$defs/upstreamRunbook'
+          kubernetesApps:
+            $ref: '#/properties/alerts/properties/runbookUrls/$defs/upstreamRunbook'
+          kubernetesResources:
+            $ref: '#/properties/alerts/properties/runbookUrls/$defs/upstreamRunbook'
+          kubernetesStorage:
+            $ref: '#/properties/alerts/properties/runbookUrls/$defs/upstreamRunbook'
+          kubernetesSystem:
+            $ref: '#/properties/alerts/properties/runbookUrls/$defs/upstreamRunbook'
+          kured:
+            $ref: '#/properties/alerts/properties/runbookUrls/$defs/noUpstreamRunbook'
+          missingMetrics:
+            $ref: '#/properties/alerts/properties/runbookUrls/$defs/noUpstreamRunbook'
+          nodeExporter:
+            $ref: '#/properties/alerts/properties/runbookUrls/$defs/upstreamRunbook'
+          nodeNetwork:
+            $ref: '#/properties/alerts/properties/runbookUrls/$defs/upstreamRunbook'
+          opensearch:
+            $ref: '#/properties/alerts/properties/runbookUrls/$defs/noUpstreamRunbook'
+          openstack:
+            $ref: '#/properties/alerts/properties/runbookUrls/$defs/noUpstreamRunbook'
+          packetsDropped:
+            $ref: '#/properties/alerts/properties/runbookUrls/$defs/noUpstreamRunbook'
+          prometheusOperator:
+            $ref: '#/properties/alerts/properties/runbookUrls/$defs/upstreamRunbook'
+          prometheus:
+            $ref: '#/properties/alerts/properties/runbookUrls/$defs/upstreamRunbook'
+          thanos:
+            title: Runbooks for thanos alerts
+            description: |-
+              Example:
+
+              group: link-to-alert-group-runbook
+              AlertName: link-to-specific-alert-runbook
+
+              Uses upstream runbooks by default
+
+              https://github.com/thanos-io/thanos/tree/main/mixin/runbook.md
+            type: object
```

Comment:
Here too?

```suggestion
            type: object
            additionalProperties:
              type: string
```

Reply:
Right I forgot about that, I assume I should define group there as well